geniza/common/utils.py

```diff
@@ -76,3 +76,18 @@ def custom_tag_string(tag_string):
 
 def simplify_quotes(text):
     return text.translate(smart_quote_conversion)
+
+
+import time
+
+TIMENOW = time.time()
+
+
+def timeprint(*x, **y):
+    global TIMENOW
+    now = time.time()
+    print(f"[+{now-TIMENOW:.3f}s]", *x, **y)
+    TIMENOW = now
+
```

Comment:
If you want to keep the duration information (which I think would probably be helpful for testing/optimizing), I recommend putting that information in debug logging. Here's an example of how I've done that in the past:

https://github.com/Princeton-CDH/geniza/blob/main/geniza/annotations/signals.py#L73-L78